src/jit/lsra.cpp

```diff
@@ -1755,12 +1756,12 @@ void LinearScan::identifyCandidates()
     unsigned int floatVarCount = 0;
     unsigned int thresholdFPRefCntWtd = 4 * BB_UNITY_WEIGHT;
     unsigned int maybeFPRefCntWtd = 2 * BB_UNITY_WEIGHT;
-#ifdef FEATURE_SIMD
+#if FEATURE_PARTIAL_SIMD_CALLEE_SAVE
     VarSetOps::AssignNoCopy(compiler, largeVectorVars, VarSetOps::MakeEmpty(compiler));
     VarSetOps::AssignNoCopy(compiler, largeVectorCalleeSaveCandidateVars, VarSetOps::MakeEmpty(compiler));
     unsigned int largeVectorVarCount = 0;
     unsigned int thresholdLargeVectorRefCntWtd = 4 * BB_UNITY_WEIGHT;
-#endif // FEATURE_SIMD
+#endif // FEATURE_PARTIAL_SIMD_CALLEE_SAVE
```

Comment:
Before we enable AVX2 SIMD intrinsics by default, we should measure whether any existing SIMD benchmarks would benefit using AVX2 compared to SSE2.  If saving and restoring of AVX2 introduces a significant perf penalty and offsets all perf benefits, we should consider not enabling it by default on unix.

How mandelbrot and raytracer perf number compare: scalar implementation vs SSE2 VS AVX2 on unix?